tests/Aspire.Hosting.Tests/Utils/ArgumentEvaluator.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Aspire.Hosting.Tests.Utils;
 
-internal sealed class ArgumentEvaluator
+public sealed class ArgumentEvaluator
```

Comment:
Make this shared source.

Reply:
We need a bunch of stuff from Aspire.Hosting.Tests, making this shared source would make things more difficult.